client/dashboard/sites/features.ts

```diff
@@ -1,52 +1,95 @@
 import { DotcomFeatures } from '../data/constants';
-import {
-	hasPlanFeature,
-	hasHostingFeatures,
-	hasAdvancedHostingFeatures,
-} from '../utils/site-features';
+import { hasPlanFeature } from '../utils/site-features';
 import type { Site, User } from '../data/types';
 
+export const HostingFeatures = {
+	SETTINGS_PHP: DotcomFeatures.SFTP,
```

Comment:
Do we need the `prefix`? It seems the value would be the same no matter where it is 🤔

Reply:
Sorry what prefix? If you're talking about `SETTINGS_`, there will be `DEPLOYMENTS`, etc. in the future.

Reply:
Yes, I’m talking about `SETTINGS_`. Even if we have other prefixes like `DEPLOYMENTS`, what’s the difference between `SETTINGS_PHP` and `DEPLOYMENTS_PHP` (just as an example)? I think they're still the same, right 🤔

Reply:
Ah no no, I mean we will have

- `SETTINGS_PHP`
- `DEPLOYMENTS`
- `MONITORING`
- `LOGS`
etc (based on the location in the UI tab) 😄

Reply:
They are just names/identifiers which represent the feature. It's like the `settings` part in `canViewCachingSettings()`, `settings-caching/` directory, route names, etc... 😄